### ISSUE TITLE
[Comments] - Home Manager: dotfiles management

### ISSUE THREAD

**Issue by gvolpe** [gvolpe/blog#32]

**Comment by bestlem** [comment]
Can you make the page readable - the code and highlighted words are readable but the rest is just not.

There needs to be contrast.

Even If I switch to a dark mode it is no better. The text just does not standout from the background.

**Comment by bestlem** [comment]
All your text is unreadable - having a mid grey background does not differentiate between the text and the background.

Test is white on black or black on white for a reason - same grey can be used for different sections but the grey needs to be very near the main background.

The Atom feed is easy to read. However I normally would only set that up after seeing if the article was worth reading. I have hundreds of feeds - most now dead.

**Comment by gvolpe** [comment]
I can read that perfectly, and my vision is reduced, might not be for you then :)

**Comment by gvolpe** [comment]
Deleting comments unrelated to this blog post, please refer to https://github.com/gvolpe/gvolpe.github.io/issues/3

**Comment by zodman** [comment]
Thanks for the information, you are using the stable version of the home manager right ?

**Comment by gvolpe** [comment]
@zodman no, I track the [master branch](https://github.com/gvolpe/nix-config/blob/2752caa10ebef78d0c41bf950e9d606e8505f5db/flake.nix#L38), but all that is mentioned in this post should be available in stable versions too.

**Comment by zodman** [comment]
> @zodman no, I track the [master branch](https://github.com/gvolpe/nix-config/blob/2752caa10ebef78d0c41bf950e9d606e8505f5db/flake.nix#L38), but all that is mentioned in this post should be available in stable versions too.

master branch points to an unstable nix channel, mmm Im implemented that function for managing dotfiles and I giveup and manage it with stow ... I will try again ...

**Comment by gvolpe** [comment]
@zodman I highly recommend moving over to nix flakes and ditch channels. It's the easiest way to reason about things, channels were a mistake... 😬 Don't give up, there's light at the end of the tunnel 😄

**Comment by zodman** [comment]
error related: https://github.com/nix-community/home-manager/issues/4692
